apps/next/src/app/app.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { TestBed, async } from '@angular/core/testing';
-import { AppComponent } from './app.component';
+import { AppComponent } from './app';
```

Comment:
I think this import is wrong - there is no AppComponent inside app.ts
Change to Next